public/components/loginForm.jsx

```diff
@@ -1,14 +1,38 @@
+'use client';
+
+import React, { useState } from 'react';
 import css from '../css/login.module.css';
+
 export default function LoginForm() {
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleUsernameChange = (e) => {
+        const value = e.target.value;
+        if (/^[A-Za-z0-9]*$/.test(value)) {
+            setUsername(value);
+        }
+    };
+
+    const handlePasswordChange = (e) => {
+        const value = e.target.value;
+        if (/^[A-Za-z0-9]*$/.test(value)) {
+            setPassword(value);
+        }
+    };
+
+    const saveduUsername = username;
+    const savedPassword = password;
+    
     return (
         <div className={css.componentWrapper}>
             <div className={css.blurryBackground}></div>
             <div className={css.formWrapper}>
                 <form>
                     <h1>L<span><img src="https://cdn-icons-png.flaticon.com/128/868/868596.png" /></span>gin</h1>
                     <section className={css.inputs}>
-                        <input type="text" id="_username" placeholder="username..." />
-                        <input type="password" id="_password" placeholder="password..." />
+                        <input type="text" id="_username" placeholder="username..." onChange={handleUsernameChange} value={username}/>
+                        <input type="password" id="_password" placeholder="password..." onChange={handlePasswordChange} value={password}/>
```

Comment:
dont know why, there is an eye symbol to show the password value while entering input to the input field. remove it.